### ISSUE TITLE
Resolve Issue with Application List in CI

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#35488]
# Description

* The previous iteration of the app list is omitting certain products.
* Starting 1/12, the code generating the app list has stopped working and passing the list between jobs.
* Debug app list failure with other CI stakeholders

**Comment by CBonade** [comment]
This has been merged in and solved! 

Something changed on GitHub's side that was causing curly braces to be flagged as sensitive characters and encoded as asterisks. We solved this by switching our array of objects to an array of arrays, and converting it to objects right before it goes to BIgQuery.

**Comment by pjhill** [comment]
Clean up -- 
- [ ] Gather feedback from GitHub SMEs
- [ ] Write up a post-mortem document for what went wrong and how and why the fix works

**Comment by CBonade** [comment]
This has been resolved. The source of the issue ended up being a combination of how GitHub processes secrets, and how our secret was entered in the AWS parameter store.  GitHub reads the secrets line by line and matches those lines as sensitive characters. We had a secret that was non-minified JSON, which means that curly braces were on lines by themselves. As such, each curly brace was flagged as a match for sensitive characters. I replaced the secret in the AWS param store with a minified version and it corrected the issue. 

I've updated the documentation to explain this here: https://vfs.atlassian.net/wiki/spaces/pilot/pages/1745911823/GitHub+Actions+usage+for+VA.gov+frontend+engineers
